### PR TITLE
Middle arrow flips input/output for swap/send

### LINKED ISSUE #147: Clicking center arrow in swap and send should flip input and output tokens

### PR DESCRIPTION
**Pull request by kennyt**
Closes #147

### CODE REVIEW

**Comment by kennyt** [comment]
@chikeichan do I have to do anything with `inputAmountB` for send/swap for this?

**Comment by haydenadams** [comment]
@kennyt technically there can be a very very small price difference between swap input and swap output do to integer rounding. It probably doesn't matter (output is already an estimate) but you could recalculate output using the old output as an input.

**Comment by ChristophSiegenthaler** [comment]
Shouldn't the "estimated" remark remain on the output field instead of switching to the input field?

**Comment by haydenadams** [comment]
Actually thats true. If you enter an exact input and click the arrow then the old input should become an exact output.

**Comment by kennyt** [comment]
@haydenadams that's what's happening in the gif right? the `.02` is the exact amount and it stays that way after switching

**Comment by haydenadams** [comment]
@kennyt you're right sorry, confusing myself now

**Comment by ChristophSiegenthaler** [comment]
Ah, that's true. And if one changes the input value again only the "(estimated)" switches to the output field as it should be. Nice! Sorry for the confusion :)

File: src/pages/Send/index.js [hunk 1]
```diff
@@ -100,6 +100,17 @@ class Send extends Component {
...
   }
 
+  flipInputOutput = () => {
+    const { state } = this;
+    this.setState({
+      inputValue: state.outputValue,
+      outputValue: state.inputValue,
+      inputCurrency: state.outputCurrency,
+      outputCurrency: state.inputCurrency,
+      lastEditedField: state.lastEditedField === INPUT ? OUTPUT : INPUT
```
Comment (chikeichan) [inline src/pages/Send/index.js]: - `lastEditedFied` should stay the same
- `inputCurrency` and `outputCurrency` should switch
- if `lastEditedField` is `inputValue`, `inputValue` should be `outputValue`, and `outputValue` should be `undefined` or `0`. 
- ^ vice versa

Then recalcForm will take care of the rest.

File: src/pages/Swap/index.js [hunk 2]
```diff
@@ -97,6 +97,17 @@ class Swap extends Component {
...
   }
 
+  flipInputOutput = () => {
+    const { state } = this;
+    this.setState({
+      inputValue: state.outputValue,
+      outputValue: state.inputValue,
+      inputCurrency: state.outputCurrency,
+      outputCurrency: state.inputCurrency,
+      lastEditedField: state.lastEditedField === INPUT ? OUTPUT : INPUT
```
Comment (chikeichan) [inline src/pages/Swap/index.js]: see above

**Pull request merged**